docs/source/api/rainmaker.rst

```diff
@@ -139,6 +139,26 @@ This function will return
 1. `ESP_OK` : On success
 2. Error in case of failure
 
+RMaker.enableSystemService
+**************************
+
+This API enables the System service for the node. It should be called after `RMaker.initNode()` and before `RMaker.start()`.
+For more information, check `here <https://rainmaker.espressif.com/docs/sys-service.html>`__.
+
+.. code-block:: arduino
+
+    esp_err_t enableSystemService(uint16_t flags, int8_t reboot_seconds, int8_t reset_seconds, int8_t reset_reboot_seconds)
+
+* ``flags`` : Logical OR of system service flags (SYSTEM_SERV_FLAG_REBOOT, SYSTEM_SERV_FLAG_FACTORY_RESET, SYSTEM_SERV_FLAG_WIFI_RESET) as required or SYSTEM_SERV_FLAGS_ALL. 
+* ``reboot_seconds`` Time in seconds after which the device should reboot. Recommended value: 2
+* ``reset_seconds`` Time in seconds after which the device should reset(Wi-Fi or Factory). Recommended value: 2
+* ``reset_reboot_seconds`` Time in seconds after which the device should reboot after it has been reset. Value of zero would mean that there won't be any reboot after the reset. Recommended value: 2
```

Comment:
Maybe you can rephrase it like this: "Zero as a value would mean there won't be any reboot after the reset."